File: week02_Arrays/scores.c
#include <cs50.h>
#include <stdio.h>

const int N = 3;

float average(int length, int array []);

int main(void)
{
    int scores[N];
    // score[0] = 72;
    // score[1] = 73;
    // score[2] = 33;

    // scores[0] = get_int("Score: ");
    // scores[1] = get_int("Score: ");
    // scores[2] = get_int("Score: ");

    for (int i = 0; i < N; i++)
    {
        scores[i] = get_int("Score: ");
    }

    // printf("Average: %f\n", (scores[0] + scores[1] +scores[2]) / (float) 3);
    printf("Average: %f\n", average(N, scores));
}

float average(int length, int array [])
{
    int sum = 0;
    for (int i = 0; i < length; i++)
    {
        sum += array[i];
    }
    return sum / (float) length;
}

File: week06_Python/scores.py
from cs50 import get_int

#=============== THIS EQUAL ==============

# scores = [72, 73, 33]

# average = sum(scores) / len(scores)
# print(f"Average: {average}")

#=============== THIS EQUAL ==============

scores = []  #OR==>  scores = list()

for i in range(3):
    score = get_int("Scores: ")
    scores.append(score) #OR ===> scores += [score]

average = sum(scores) / len(scores)
print(f"Average: {average}")
